apps/server/src/config/server.ts

```diff
@@ -39,6 +39,12 @@ export const defaultAppConfig: AppConfig = {
 		audit: {
 			enabled: getBoolean(process.env.AUDIT_ENABLED, true),
 		},
+		recordHierarchy: {
+			nestedRecordPrefix: process.env.NESTED_RECORD_PREFIX || '',
+			nestedRecordSuffix: process.env.NESTED_RECORD_SUFFIX || 's',
```

Comment:
thought you'd have taken the pluraliser approach instead?

Reply:
This comment will be handled in PR[#105](https://github.com/overture-stack/lyric/pull/105)
Adding [plur](https://www.npmjs.com/package/plur) to pluralize words, it gave better results than [pluralize](https://www.npmjs.com/package/pluralize)